H_ecto-models.md

```diff
@@ -517,6 +517,77 @@ end
 
 That's the end of our walk-through of Ecto usage in our controller actions. There is quite a bit more that Ecto models can do. Please take a look at the [Ecto documentation](http://hexdocs.pm/ecto/) for the rest of the story.
 
+### Data Relationship and Dependencies
+
+Handling individual tables is great, but if we want to build a modern web application, we will need a way to relate our data to each other. If we were Ruby on Rails refugees, we'll be glad to know Ecto provides a very familiar API for building relationships between models:
```

Comment:
I think "refugees" is an emotionally charged word we should be careful with. 

What would you think about changing the first part of that sentence to this?
Those of us with experience using ActiveRecord will be glad to see that Ecto provides . . .